compiler/rustc_parse_format/src/lib.rs

```diff
@@ -367,23 +383,36 @@ impl<'a> Parser<'a> {
         None
     }
 
-    fn to_span_index(&self, pos: usize) -> InnerOffset {
-        let mut pos = pos;
-        // This handles the raw string case, the raw argument is the number of #
-        // in r###"..."### (we need to add one because of the `r`).
-        let raw = self.style.map_or(0, |raw| raw + 1);
-        for skip in &self.skips {
-            if pos > *skip {
-                pos += 1;
-            } else if pos == *skip && raw == 0 {
-                pos += 1;
+    fn remap_pos(&self, mut pos: usize) -> usize {
```

Comment:
Could you add a comment: what are the input and output `usize`s?
Should we introduce some type to differentiate "raw" from "remapped" positions?

Reply:
Yes, that should be `InnerOffset`, my bad.